framework/src/task_manager/task_manager_permission.c

```diff
@@ -32,7 +31,7 @@ bool taskmgr_is_permitted(int handle, pid_t pid)
 	int builtin_cnt;
 	int permission = TM_PERMISSION(handle);
 
-	builtin_cnt = get_builtin_list_cnt();
+	builtin_cnt = builtin_list_cnt;
```

Comment:
Similarly, here as well.
The two variable names in use are fairly close to each other.

Reply:
Done

Reply:
Done.